dist/install.sh

```diff
@@ -1,11 +1,12 @@
-#!/bin/sh
-#
+#!/usr/bin/env bash
```

Comment:
change filename to .bash / or no extension?

Reply:
Extension doesn't matter for shell scripts. However will remove it, as it is redundant information.

Reply:
right, though i believe prettier makes assumptions based an the sh extension vs no extension. anyway, +1 for no extension